contrib/babelfishpg_tsql/src/guc.c

```diff
@@ -605,6 +606,16 @@ define_custom_variables(void)
 				 GUC_NO_SHOW_ALL,
 				 NULL, NULL, NULL);
 
+	/* temporary GUC for enable or disable windows login */
+	DefineCustomBoolVariable("babelfishpg_tsql.allow_windows_login",
+				 gettext_noop("GUC for enable or disable windows login"),
+				 NULL,
+				 &allow_windows_login,
```

Comment:
I meant no need for pltsql with GUC name, variable should have prefix to avoid clash with similar name on engine side in future

Reply:
Done!